zenith/src/main.rs

```diff
@@ -456,24 +479,42 @@ fn handle_tenant(tenant_match: &ArgMatches, env: &local_env::LocalEnv) -> Result
     Ok(())
 }
 
-fn handle_branch(branch_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<()> {
+fn handle_timeline(timeline_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<()> {
     let pageserver = PageServerNode::from_env(env);
 
-    let tenantid = get_tenantid(branch_match, env)?;
-
-    if let Some(branchname) = branch_match.value_of("branchname") {
-        let startpoint_str = branch_match
-            .value_of("start-point")
-            .context("Missing start-point")?;
-        let branch = pageserver.branch_create(branchname, startpoint_str, &tenantid)?;
-        println!(
-            "Created branch '{}' at {:?} for tenant: {}",
-            branch.name, branch.latest_valid_lsn, tenantid,
-        );
-    } else {
-        // No arguments, list branches for tenant
-        let branches = pageserver.branch_list(&tenantid)?;
-        print_branches_tree(branches)?;
+    match timeline_match.subcommand() {
+        Some(("list", list_match)) => {
+            let tenant_id = get_tenant_id(list_match, env)?;
+            let timelines = pageserver.timeline_list(&tenant_id)?;
+            print_timelines_tree(timelines)?;
+        }
+        Some(("create", create_match)) => {
```

Comment:
Note that now external API users will be able to create ancestor-less timelines (before these changes, only `init` operation could implicitly create such timelines).

I would also imagine, that we can change the API so that there's `list`, `create` and `branch` parts in it, to add more explicitness? If so, should the same separation be done in the http API?

Reply:
I think it is pretty natural to have that feature. We can bring this up in a cloud meeting, because it may affect the way how console works now